common/script/script-api/src/main/java/org/thingsboard/script/api/Coordinates.java

```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.thingsboard.rule.engine.geo;
+package org.thingsboard.script.api;
```

Comment:
Sorry, let's actually move geo to common/util. Don't forget to create a separate package